### PR TITLE
Added method to get appLinks for a given user

### PR DESCRIPTION
**Pull request by danieldelhoyo**
Addresses issue #28

### CODE REVIEW

**Comment by jmelberg-okta** [comment]
Hi @danieldelhoyo! Thanks for the pull request. 

Before we can move forward with reviewing/merging your PR, please take a moment to read and sign a Contributor License Agreement (CLA).

Please visit [developer.okta.com/cla](http://developer.okta.com/cla/) to sign.

Then, reply here (“`Signed`”) so we can proceed. Thanks!

**Comment by danieldelhoyo** [comment]
Signed

**Comment by jmelberg-okta** [comment]
@danieldelhoyo: Can you please email us a signed copy at: [CLA@okta.com](mailto:cla@okta.com)? 

Once we receive this copy, we will begin the review.

Thanks!

**Comment by danieldelhoyo** [comment]
@jmelberg-okta just did. I understand you need to have a CLA, but the process currently defined for doing so is pretty annoying for contributors IMHO (just check the number of idling PRs you have)
I know legal stuff is hard, but as a personal comment I would suggest you guys to revisit it, it's just too tempting to keep my fork and never contribute back.

**Comment by tbelote-okta** [comment]
Agreed we are working on integrating a CLA bot so you can sign with your github account.

File: tests/SerializationTest.py [hunk 1]
```diff
@@ -190,4 +191,38 @@ def test_embedded_deserialization(self):
...
+            "hidden": true,
+            "sortOrder": 1
+          }]
+        '''
+
+        result = Utils.deserialize(json_str, AppLinks)
+
+        self.assertEqual(len(result), 2, "Cannot deserialize nested lists of objects")
+        serialized = json.dumps(result, cls=Serializer)
+        self.assertTrue("1od6bbdxlpb1BgDTi1h7" in serialized, "Id attribute isn't deserialized properly")
```
Comment (jmelberg-okta) [inline tests/SerializationTest.py]: Since we are deserializing the list, lets change this message to: `Nested lists aren't serialized properly`

Comment (jmelberg-okta) [inline tests/SerializationTest.py]: + newline

File: okta/models/user/AppLinks.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        "linkUrl": str,
+        "logoUrl": str,
+        "sortOrder": int
+    }
+
+    def __init__(self):
+
+        self.appAssignmentId = None  # str
+
+        self.appAInstanceId = None  # str
```
Comment (jmelberg-okta) [inline okta/models/user/AppLinks.py]: Spelling: `self. appAInstanceId ` -> `self.appInstanceId`

File: okta/UsersClient.py [hunk 3]
```diff
@@ -42,6 +43,17 @@ def get_user(self, uid):
...
 
+    def get_user_applinks(self, uid):
+        """Get applinks of a single user
+
+        :param uid: the user id or login
+        :type uid: str
+        :rtype: AppLinks
+        """
+        response = ApiClient.get_path(self, '/{0}/appLinks'.format(uid))
+        print response.text
```
Comment (jmelberg-okta) [inline okta/UsersClient.py]: Lets remove this `print` statement.

**Comment by danieldelhoyo** [comment]
@jmelberg-okta thanks for your comments! Please double check :)

**Comment by jmelberg-okta** [comment]
@danieldelhoyo: Looks good! Can you squash the three commits into one? We can move forward with merging it then.

**Comment by danieldelhoyo** [comment]
cancelled in favor of  #33

**Pull request closed without merging**